.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "printWidth": 120,
-  "singleQuote": true
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2
```

Comment:
Obligatory spaces vs tabs comment.

Reply:
I'm for sure team tabs but when I set it to true it indented the code all the way to Europe. Maybe it should be true with tabWidth "1"?

Reply:
I'm typically Team Tabs as well, particularly for publicly shared code and namely for accessibility reasons. The custom tabwidth settings in editors are easier for people who use screen magnifiers (who might want to set a small tabwidth so their own tabs don't indent all the way to Europe). Additionally, each tab and/or space takes up a character in a braille reader, which usually has limited room, so one tab tends to be more economical for braille reader users than 2, 4, 8, whatever number of spaces.

It's a pity that GitHub's tabwidth is 8, though.

Reply:
makes sense. Will fix this before merging!

Reply:
i'd be interested in working in a tabs project.
i've always liked tabs philosophically but have alway worked in Ruby projects where `two spaces` like a god